src/loaders/BufferGeometryLoader.js

```diff
@@ -227,15 +227,7 @@ class BufferGeometryLoader extends Loader {
 
 		if ( boundingSphere !== undefined ) {
 
-			const center = new Vector3();
-
-			if ( boundingSphere.center !== undefined ) {
-
-				center.fromArray( boundingSphere.center );
-
-			}
-
-			geometry.boundingSphere = new Sphere( center, boundingSphere.radius );
```

Comment:
I'm not sure what this condition was for since there's no comment but it doesn't seem like it's needed?

Reply:
Yes, it looks redundant.